dev/tidb-in-kubernetes/maintain/lightning.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Restore data into TiDB in Kubernetes
+summary: Learn how to quickly restore data into a TiDB cluster in Kubernetes with TiDB Lightning.
+category: how-to
+---
+
+# Restore data into TiDB in Kubernetes
+
+This document describes how to restore data into a TiDB cluster in Kubernetes using [TiDB Lightning](https://github.com/pingcap/tidb-lightning).
+
+TiDB Lightning contains two components: tidb-lightning and tikv-importer. In Kubernetes, the tikv-importer is inside the Helm chart of the TiDB cluster, it is deployed as a `StatefulSet` with `replicas=1` while tidb-lightning is in a separate Helm chart and deployed as a `Job`.
```

Comment:
Briefly describe how they work? Then they may understand why these two components should be deployed in this way.

Reply:
It can not be described easily. The tidb-lightning supports both server-mode and non server-mode, it can be deployed both ways in k8s. Also the tikv-importer can be in a separate chart. It's not right or wrong choice, just preferable way made by us.